README.md

```diff
@@ -6,23 +6,15 @@
 [`master`](https://github.com/itisfoundation/osparc-simcore/tree/master)
 [![Requires.io]](https://requires.io/github/ITISFoundation/osparc-simcore/requirements/?branch=master "State of third party python dependencies")
 [![travis-ci]](https://travis-ci.org/ITISFoundation/osparc-simcore "State of CI: build, test and pushing images")
-[![coverals.io]](https://coveralls.io/github/ITISFoundation/osparc-simcore?branch=master "Test coverage")
+[![Coverage Status](https://coveralls.io/repos/github/ITISFoundation/osparc-simcore/badge.svg?branch=master)](https://coveralls.io/github/ITISFoundation/osparc-simcore?branch=master)

```

Comment:
like I commented in @GitHK [PR](https://github.com/ITISFoundation/osparc-simcore/pull/1481) we also use codecov.io (from github-ci). We might have to choose or put both.